src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,140 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    static class Node<T> {
+        private T element;
+        private Node<T> next;
```

Comment:
please add also `private Node<T> prev;` and implement all needed logic, linked list is a two-directional list.